src/lib.rs

```diff
@@ -1950,6 +1950,8 @@ impl Build {
             cmd.push_cc_arg(warnings_to_errors_flag);
         }
 
+       disable_localization(&mut cmd);
```

Comment:
Just so that we've considered it: Should this be a "default" flag (should it be added in `add_default_flags`)?

My own opinion is no, since `CRATE_CC_NO_DEFAULTS` seems more targeted towards making cross-compilation work.

Reply:
I think since it doesn't have any impact on the artifact generated, it shouldn't be considered as a flag